### DISCUSSION TITLE
Jogging is not working properly

### DISCUSSION THREAD

**Discussion by Cheese1663** [arkypita/LaserGRBL#1485] [Q&A]
I have purchased the Yorahome 6550 with 40w laser. When i hit the arrows on the manual jog it will not travel straight up and down, it travels diagonally. It will not travel straight right or left at all. Is this a software issue or a machine issue?

**Comment by gmmanonymus111** [comment]
It's hard to tell yet. Does this `Yorahome 6550` use GRBL firmware? If it does, which version? (What does LaserGRBL say when connected to your machine? [eg. `GRBL v1.1f`])

**Comment by arkypita** [comment]
Not a LaserGRBL issue.
Probably is hardware wiring or firmware configuration issue.
